db/structure.sql

```diff
@@ -1488,11 +1133,21 @@ CREATE TABLE `users` (
   KEY `index_users_on_uid` (`uid`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci ROW_FORMAT=DYNAMIC;
 /*!40101 SET character_set_client = @saved_cs_client */;
-
---
--- Table structure for table `watchers`
---
-
+DROP TABLE IF EXISTS `versions`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `versions` (
+  `id` int(11) NOT NULL AUTO_INCREMENT,
+  `item_type` varchar(191) NOT NULL,
+  `item_id` bigint(20) NOT NULL,
+  `event` varchar(255) NOT NULL,
+  `whodunnit` varchar(255) DEFAULT NULL,
+  `object` longtext,
+  `created_at` datetime DEFAULT NULL,
+  PRIMARY KEY (`id`),
+  KEY `index_versions_on_item_type_and_item_id` (`item_type`,`item_id`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
```

Comment:
The `versions` table for Papertrail should not be added as part of this release.